source/agora/flash/Channel.d

```diff
@@ -777,6 +777,38 @@ LOuter: while (1)
             auto settle_tx = this.channel_updates[$ - 1].settle_tx;
             // point the input to the last update utxo
             settle_tx.inputs[0].utxo = this.last_externalized_update_utxo;
+            auto settle_input = settle_tx.inputs[0];
```

Comment:
Settle txs can't have fees added to them this way. They are not the same with the update TXs. Let's drop this commit from this PR and discuss it on another one after merging the others.

Reply:
Will discuss it next time.